src/systems/physics/Physics.cc

```diff
@@ -732,6 +736,18 @@ void Physics::Configure(const Entity &_entity,
         [](const std::string &_a, const std::string &_b){return _a == _b;});
   }
 
+  // Check if entity names should be populated in contact points.
+  if (_sdf->HasElement("contacts"))
+  {
+    auto sdfClone = _sdf->Clone();
+    auto contactsElement = sdfClone->GetElement("contacts");
+    if (contactsElement->HasElement("include_entity_names"))
+    {
+      this->dataPtr->contactsEntityNames = contactsElement->Get<bool>(
+        "include_entity_names");
+    }
+  }
```

Comment:
This is an old pattern. This can now be simplified as
```suggestion
  auto contactsElement = _sdf->FindElement("contacts");
  if (contactsElement)
  {
    this->dataPtr->contactsEntityNames = contactsElement->Get<bool>(
      "include_entity_names", true);  
  }
```

Reply:
Done